DevCycle.SDK.Server.Common/Model/Local/Variable.cs

```diff
@@ -156,5 +165,32 @@ public static TypeEnum DetermineType(T variableValue)
                 throw e;
             }
         }
+
+        public static TypeEnum DetermineType(string type)
```

Comment:
don't we already have a `DetermineType` method in here that uses the actual variable value to figure out the correct type? We should be using that, it seems like this is just trying to convert strings into the same strings that are contained in the TypeEnum?

Reply:
we do, but also we have a string that tells us the type coming from the server, we should just use that instead of trying to determine the type manually

Reply:
the type of the variable should always match what the SDK has detected the type to be (based on the data type of the default value). It should never be possible for the variable instance to have a different "type" than what the actual type is. Basically, the server's value for that field should just be ignored IMO.